src/components/Header/Search/Text/Text.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import styles from "./text.module.css";
+
+function Text() {
```

Comment:
Components that are used in multiple places should be inside the common folder

The same with names